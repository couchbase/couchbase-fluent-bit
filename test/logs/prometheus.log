level=info ts=2021-07-21T15:54:43.568Z caller=main.go:356 msg="Starting Prometheus" version="(version=2.22.0, branch=HEAD, revision=a6239a377d49104ac7253a99aef8feb8dee0a7c2)"
level=info ts=2021-07-21T15:54:43.568Z caller=main.go:361 build_context="(go=go1.14.2, user=couchbase@d0e641caddb8, date=20210325-16:51:45)"
level=info ts=2021-07-21T15:54:43.568Z caller=main.go:362 host_details="(Linux 5.10.25-linuxkit #1 SMP Tue Mar 23 09:27:39 UTC 2021 x86_64 0fa163ea7e3d (none))"
level=info ts=2021-07-21T15:54:43.568Z caller=main.go:363 fd_limits="(soft=1048576, hard=1048576)"
level=info ts=2021-07-21T15:54:43.568Z caller=main.go:364 vm_limits="(soft=unlimited, hard=unlimited)"
level=info ts=2021-07-21T15:54:43.569Z caller=web.go:488 component=web msg="Loading Basic Auth file" filename=/opt/couchbase/var/lib/couchbase/config/prometheus_auth
level=info ts=2021-07-21T15:54:43.570Z caller=main.go:715 msg="Starting TSDB ..."
level=info ts=2021-07-21T15:54:43.571Z caller=web.go:619 component=web msg="Start listening for connections" address=127.0.0.1:9123
level=info ts=2021-07-21T15:54:43.575Z caller=head.go:642 component=tsdb msg="Replaying on-disk memory mappable chunks if any"
level=info ts=2021-07-21T15:54:43.575Z caller=head.go:656 component=tsdb msg="On-disk memory mappable chunks replay completed" duration=2.146µs
level=info ts=2021-07-21T15:54:43.575Z caller=head.go:714 component=tsdb msg="WAL segment loaded" segment=0 maxSegment=0
level=info ts=2021-07-21T15:54:43.575Z caller=head.go:719 component=tsdb msg="WAL replay completed" checkpoint_replay_duration=34.26µs wal_replay_duration=531.682µs total_replay_duration=581.144µs
level=info ts=2021-07-21T15:54:43.576Z caller=main.go:735 fs_type=EXT4_SUPER_MAGIC
level=info ts=2021-07-21T15:54:43.576Z caller=main.go:738 msg="TSDB started"
level=debug ts=2021-07-21T15:54:43.577Z caller=main.go:739 msg="TSDB options" MinBlockDuration=2h MaxBlockDuration=1d1h MaxBytes=1GiB NoLockfile=true RetentionDuration=1y WALSegmentSize=0B AllowOverlappingBlocks=false WALCompression=true
level=debug ts=2021-07-21T15:54:43.578Z caller=manager.go:195 component="discovery manager scrape" msg="Starting provider" provider=static/2 subs=[kv_high_cardinality]
level=debug ts=2021-07-21T15:54:43.578Z caller=manager.go:213 component="discovery manager scrape" msg="Discoverer channel closed" provider=static/0
level=info ts=2021-07-21T15:54:43.579Z caller=main.go:895 msg="Completed loading of configuration file" filename=/opt/couchbase/var/lib/couchbase/config/prometheus.yml totalDuration=2.468829ms remote_storage=1.529µs web_handler=58.799µs query_engine=1.191µs scrape=503.071µs scrape_sd=674.205µs notify=1.154µs notify_sd=3.711µs rules=746.829µs
level=debug ts=2021-07-21T15:54:44.456Z caller=manager.go:195 component="discovery manager scrape" msg="Starting provider" provider=static/0 subs=[ns_server_high_cardinality]
level=debug ts=2021-07-21T15:54:44.457Z caller=manager.go:213 component="discovery manager scrape" msg="Discoverer channel closed" provider=static/5
level=info ts=2021-07-21T15:54:44.457Z caller=main.go:895 msg="Completed loading of configuration file" filename=/opt/couchbase/var/lib/couchbase/config/prometheus.yml totalDuration=2.983761ms remote_storage=2.765µs web_handler=358.528µs query_engine=1.453µs scrape=60.125µs scrape_sd=186.89µs notify=1.713µs notify_sd=3.28µs rules=613.834µs
level=debug ts=2021-07-21T15:54:54.493Z caller=scrape.go:1117 component="scrape manager" scrape_pool=cbas_high_cardinality target=http://127.0.0.1:8095/_prometheusMetricsHigh msg="Scrape failed" err="Get \"http://127.0.0.1:8095/_prometheusMetricsHigh\": dial tcp 127.0.0.1:8095: connect: connection refused"
