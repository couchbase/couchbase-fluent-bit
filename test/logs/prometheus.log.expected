/fluent-bit/test/logs/prometheus.log.actual: [1626882883.568000000, {"file_path":"/fluent-bit/test/logs","file":"prometheus.log","filename":"/fluent-bit/test/logs/prometheus.log","timestamp":"2021-07-21T15:54:43.568Z","caller":"main.go:356","message":"Starting Prometheus","version":"(version=2.22.0, branch=HEAD, revision=a6239a377d49104ac7253a99aef8feb8dee0a7c2)","hostname":"test-container","logshipper":"couchbase.sidecar.fluentbit","fluentbit.version":"99","pod":{"name":"test-pod"},"couchbase":{"logging.version":"99","cluster":"test-cluster"},"level":"INFO"}]
/fluent-bit/test/logs/prometheus.log.actual: [1626882883.568000000, {"file_path":"/fluent-bit/test/logs","file":"prometheus.log","filename":"/fluent-bit/test/logs/prometheus.log","timestamp":"2021-07-21T15:54:43.568Z","caller":"main.go:361","build_context":"(go=go1.14.2, user=couchbase@d0e641caddb8, date=20210325-16:51:45)","message":"UNKNOWN","hostname":"test-container","logshipper":"couchbase.sidecar.fluentbit","fluentbit.version":"99","pod":{"name":"test-pod"},"couchbase":{"logging.version":"99","cluster":"test-cluster"},"level":"INFO"}]
/fluent-bit/test/logs/prometheus.log.actual: [1626882883.568000000, {"file_path":"/fluent-bit/test/logs","file":"prometheus.log","filename":"/fluent-bit/test/logs/prometheus.log","timestamp":"2021-07-21T15:54:43.568Z","caller":"main.go:362","host_details":"(Linux 5.10.25-linuxkit #1 SMP Tue Mar 23 09:27:39 UTC 2021 x86_64 0fa163ea7e3d (none))","message":"UNKNOWN","hostname":"test-container","logshipper":"couchbase.sidecar.fluentbit","fluentbit.version":"99","pod":{"name":"test-pod"},"couchbase":{"logging.version":"99","cluster":"test-cluster"},"level":"INFO"}]
/fluent-bit/test/logs/prometheus.log.actual: [1626882883.568000000, {"file_path":"/fluent-bit/test/logs","file":"prometheus.log","filename":"/fluent-bit/test/logs/prometheus.log","timestamp":"2021-07-21T15:54:43.568Z","caller":"main.go:363","fd_limits":"(soft=1048576, hard=1048576)","message":"UNKNOWN","hostname":"test-container","logshipper":"couchbase.sidecar.fluentbit","fluentbit.version":"99","pod":{"name":"test-pod"},"couchbase":{"logging.version":"99","cluster":"test-cluster"},"level":"INFO"}]
/fluent-bit/test/logs/prometheus.log.actual: [1626882883.568000000, {"file_path":"/fluent-bit/test/logs","file":"prometheus.log","filename":"/fluent-bit/test/logs/prometheus.log","timestamp":"2021-07-21T15:54:43.568Z","caller":"main.go:364","vm_limits":"(soft=unlimited, hard=unlimited)","message":"UNKNOWN","hostname":"test-container","logshipper":"couchbase.sidecar.fluentbit","fluentbit.version":"99","pod":{"name":"test-pod"},"couchbase":{"logging.version":"99","cluster":"test-cluster"},"level":"INFO"}]
/fluent-bit/test/logs/prometheus.log.actual: [1626882883.569000000, {"file_path":"/fluent-bit/test/logs","file":"prometheus.log","filename":"/fluent-bit/test/logs/prometheus.log","timestamp":"2021-07-21T15:54:43.569Z","caller":"web.go:488","component":"web","message":"Loading Basic Auth file","message_filename":"/opt/couchbase/var/lib/couchbase/config/prometheus_auth","hostname":"test-container","logshipper":"couchbase.sidecar.fluentbit","fluentbit.version":"99","pod":{"name":"test-pod"},"couchbase":{"logging.version":"99","cluster":"test-cluster"},"level":"INFO"}]
/fluent-bit/test/logs/prometheus.log.actual: [1626882883.570000000, {"file_path":"/fluent-bit/test/logs","file":"prometheus.log","filename":"/fluent-bit/test/logs/prometheus.log","timestamp":"2021-07-21T15:54:43.570Z","caller":"main.go:715","message":"Starting TSDB ...","hostname":"test-container","logshipper":"couchbase.sidecar.fluentbit","fluentbit.version":"99","pod":{"name":"test-pod"},"couchbase":{"logging.version":"99","cluster":"test-cluster"},"level":"INFO"}]
/fluent-bit/test/logs/prometheus.log.actual: [1626882883.571000000, {"file_path":"/fluent-bit/test/logs","file":"prometheus.log","filename":"/fluent-bit/test/logs/prometheus.log","timestamp":"2021-07-21T15:54:43.571Z","caller":"web.go:619","component":"web","message":"Start listening for connections","address":"127.0.0.1:9123","hostname":"test-container","logshipper":"couchbase.sidecar.fluentbit","fluentbit.version":"99","pod":{"name":"test-pod"},"couchbase":{"logging.version":"99","cluster":"test-cluster"},"level":"INFO"}]
/fluent-bit/test/logs/prometheus.log.actual: [1626882883.575000000, {"file_path":"/fluent-bit/test/logs","file":"prometheus.log","filename":"/fluent-bit/test/logs/prometheus.log","timestamp":"2021-07-21T15:54:43.575Z","caller":"head.go:642","component":"tsdb","message":"Replaying on-disk memory mappable chunks if any","hostname":"test-container","logshipper":"couchbase.sidecar.fluentbit","fluentbit.version":"99","pod":{"name":"test-pod"},"couchbase":{"logging.version":"99","cluster":"test-cluster"},"level":"INFO"}]
/fluent-bit/test/logs/prometheus.log.actual: [1626882883.575000000, {"file_path":"/fluent-bit/test/logs","file":"prometheus.log","filename":"/fluent-bit/test/logs/prometheus.log","timestamp":"2021-07-21T15:54:43.575Z","caller":"head.go:656","component":"tsdb","message":"On-disk memory mappable chunks replay completed","duration":"2.146µs","hostname":"test-container","logshipper":"couchbase.sidecar.fluentbit","fluentbit.version":"99","pod":{"name":"test-pod"},"couchbase":{"logging.version":"99","cluster":"test-cluster"},"level":"INFO"}]
/fluent-bit/test/logs/prometheus.log.actual: [1626882883.575000000, {"file_path":"/fluent-bit/test/logs","file":"prometheus.log","filename":"/fluent-bit/test/logs/prometheus.log","timestamp":"2021-07-21T15:54:43.575Z","caller":"head.go:714","component":"tsdb","message":"WAL segment loaded","segment":"0","maxSegment":"0","hostname":"test-container","logshipper":"couchbase.sidecar.fluentbit","fluentbit.version":"99","pod":{"name":"test-pod"},"couchbase":{"logging.version":"99","cluster":"test-cluster"},"level":"INFO"}]
/fluent-bit/test/logs/prometheus.log.actual: [1626882883.575000000, {"file_path":"/fluent-bit/test/logs","file":"prometheus.log","filename":"/fluent-bit/test/logs/prometheus.log","timestamp":"2021-07-21T15:54:43.575Z","caller":"head.go:719","component":"tsdb","message":"WAL replay completed","checkpoint_replay_duration":"34.26µs","wal_replay_duration":"531.682µs","total_replay_duration":"581.144µs","hostname":"test-container","logshipper":"couchbase.sidecar.fluentbit","fluentbit.version":"99","pod":{"name":"test-pod"},"couchbase":{"logging.version":"99","cluster":"test-cluster"},"level":"INFO"}]
/fluent-bit/test/logs/prometheus.log.actual: [1626882883.576000000, {"file_path":"/fluent-bit/test/logs","file":"prometheus.log","filename":"/fluent-bit/test/logs/prometheus.log","timestamp":"2021-07-21T15:54:43.576Z","caller":"main.go:735","fs_type":"EXT4_SUPER_MAGIC","message":"UNKNOWN","hostname":"test-container","logshipper":"couchbase.sidecar.fluentbit","fluentbit.version":"99","pod":{"name":"test-pod"},"couchbase":{"logging.version":"99","cluster":"test-cluster"},"level":"INFO"}]
/fluent-bit/test/logs/prometheus.log.actual: [1626882883.576000000, {"file_path":"/fluent-bit/test/logs","file":"prometheus.log","filename":"/fluent-bit/test/logs/prometheus.log","timestamp":"2021-07-21T15:54:43.576Z","caller":"main.go:738","message":"TSDB started","hostname":"test-container","logshipper":"couchbase.sidecar.fluentbit","fluentbit.version":"99","pod":{"name":"test-pod"},"couchbase":{"logging.version":"99","cluster":"test-cluster"},"level":"INFO"}]
/fluent-bit/test/logs/prometheus.log.actual: [1626882883.577000000, {"file_path":"/fluent-bit/test/logs","file":"prometheus.log","filename":"/fluent-bit/test/logs/prometheus.log","timestamp":"2021-07-21T15:54:43.577Z","caller":"main.go:739","message":"TSDB options","MinBlockDuration":"2h","MaxBlockDuration":"1d1h","MaxBytes":"1GiB","NoLockfile":"true","RetentionDuration":"1y","WALSegmentSize":"0B","AllowOverlappingBlocks":"false","WALCompression":"true","hostname":"test-container","logshipper":"couchbase.sidecar.fluentbit","fluentbit.version":"99","pod":{"name":"test-pod"},"couchbase":{"logging.version":"99","cluster":"test-cluster"},"level":"DEBUG"}]
/fluent-bit/test/logs/prometheus.log.actual: [1626882883.578000000, {"file_path":"/fluent-bit/test/logs","file":"prometheus.log","filename":"/fluent-bit/test/logs/prometheus.log","timestamp":"2021-07-21T15:54:43.578Z","caller":"manager.go:195","component":"discovery manager scrape","message":"Starting provider","provider":"static/2","subs":"[kv_high_cardinality]","hostname":"test-container","logshipper":"couchbase.sidecar.fluentbit","fluentbit.version":"99","pod":{"name":"test-pod"},"couchbase":{"logging.version":"99","cluster":"test-cluster"},"level":"DEBUG"}]
/fluent-bit/test/logs/prometheus.log.actual: [1626882883.578000000, {"file_path":"/fluent-bit/test/logs","file":"prometheus.log","filename":"/fluent-bit/test/logs/prometheus.log","timestamp":"2021-07-21T15:54:43.578Z","caller":"manager.go:213","component":"discovery manager scrape","message":"Discoverer channel closed","provider":"static/0","hostname":"test-container","logshipper":"couchbase.sidecar.fluentbit","fluentbit.version":"99","pod":{"name":"test-pod"},"couchbase":{"logging.version":"99","cluster":"test-cluster"},"level":"DEBUG"}]
/fluent-bit/test/logs/prometheus.log.actual: [1626882883.579000000, {"file_path":"/fluent-bit/test/logs","file":"prometheus.log","filename":"/fluent-bit/test/logs/prometheus.log","timestamp":"2021-07-21T15:54:43.579Z","caller":"main.go:895","message":"Completed loading of configuration file","message_filename":"/opt/couchbase/var/lib/couchbase/config/prometheus.yml","totalDuration":"2.468829ms","remote_storage":"1.529µs","web_handler":"58.799µs","query_engine":"1.191µs","scrape":"503.071µs","scrape_sd":"674.205µs","notify":"1.154µs","notify_sd":"3.711µs","rules":"746.829µs","hostname":"test-container","logshipper":"couchbase.sidecar.fluentbit","fluentbit.version":"99","pod":{"name":"test-pod"},"couchbase":{"logging.version":"99","cluster":"test-cluster"},"level":"INFO"}]
/fluent-bit/test/logs/prometheus.log.actual: [1626882884.456000000, {"file_path":"/fluent-bit/test/logs","file":"prometheus.log","filename":"/fluent-bit/test/logs/prometheus.log","timestamp":"2021-07-21T15:54:44.456Z","caller":"manager.go:195","component":"discovery manager scrape","message":"Starting provider","provider":"static/0","subs":"[ns_server_high_cardinality]","hostname":"test-container","logshipper":"couchbase.sidecar.fluentbit","fluentbit.version":"99","pod":{"name":"test-pod"},"couchbase":{"logging.version":"99","cluster":"test-cluster"},"level":"DEBUG"}]
/fluent-bit/test/logs/prometheus.log.actual: [1626882884.457000000, {"file_path":"/fluent-bit/test/logs","file":"prometheus.log","filename":"/fluent-bit/test/logs/prometheus.log","timestamp":"2021-07-21T15:54:44.457Z","caller":"manager.go:213","component":"discovery manager scrape","message":"Discoverer channel closed","provider":"static/5","hostname":"test-container","logshipper":"couchbase.sidecar.fluentbit","fluentbit.version":"99","pod":{"name":"test-pod"},"couchbase":{"logging.version":"99","cluster":"test-cluster"},"level":"DEBUG"}]
/fluent-bit/test/logs/prometheus.log.actual: [1626882884.457000000, {"file_path":"/fluent-bit/test/logs","file":"prometheus.log","filename":"/fluent-bit/test/logs/prometheus.log","timestamp":"2021-07-21T15:54:44.457Z","caller":"main.go:895","message":"Completed loading of configuration file","message_filename":"/opt/couchbase/var/lib/couchbase/config/prometheus.yml","totalDuration":"2.983761ms","remote_storage":"2.765µs","web_handler":"358.528µs","query_engine":"1.453µs","scrape":"60.125µs","scrape_sd":"186.89µs","notify":"1.713µs","notify_sd":"3.28µs","rules":"613.834µs","hostname":"test-container","logshipper":"couchbase.sidecar.fluentbit","fluentbit.version":"99","pod":{"name":"test-pod"},"couchbase":{"logging.version":"99","cluster":"test-cluster"},"level":"INFO"}]
/fluent-bit/test/logs/prometheus.log.actual: [1626882894.493000000, {"file_path":"/fluent-bit/test/logs","file":"prometheus.log","filename":"/fluent-bit/test/logs/prometheus.log","timestamp":"2021-07-21T15:54:54.493Z","caller":"scrape.go:1117","component":"scrape manager","scrape_pool":"cbas_high_cardinality","target":"http://127.0.0.1:8095/_prometheusMetricsHigh","message":"Scrape failed","err":"Get \"http://127.0.0.1:8095/_prometheusMetricsHigh\": dial tcp 127.0.0.1:8095: connect: connection refused","hostname":"test-container","logshipper":"couchbase.sidecar.fluentbit","fluentbit.version":"99","pod":{"name":"test-pod"},"couchbase":{"logging.version":"99","cluster":"test-cluster"},"level":"DEBUG"}]
